action.yml

```diff
@@ -162,7 +162,7 @@ runs:
         path: ${{ inputs.base-path }}/zephyr-sdk
         key: ${{ env.SDK_FILE }}-${{ inputs.toolchains }}
 
-    - if: ${{ steps.cache-toolchain.outputs.cache-hit != 'true' }}
+    - if: ${{ steps.cache-toolchain.outputs.cache-hit != 'true' && inputs.skip-dependencies != 'true' }}
```

Comment:
this is meant to be in the previous commit right?

Reply:
It was, good catch. Fixed on latest push